django_test_suite_4.2.sh

```diff
@@ -56,6 +56,7 @@ DATABASES = {
        'NAME': "$TEST_DBNAME_OTHER",
    },
 }
+USE_TZ = False
```

Comment:
What is the default in Django 4.2? The reason for asking is that if the default in Django 4.2 is `USE_TZ = True` and we need to override it here to make all tests pass, then users should also override this in their settings. That again should clearly be called out in the release notes.
Or is there any way that we can automatically override this default in our Django implementation?

Reply:
Default is False in 4.2 and it would be changed to True in 5.0 https://docs.djangoproject.com/en/4.2/topics/i18n/timezones/

We need to put it explicitly here as tests were failing otherwise with a warning